services/QuillLMS/client/app/bundles/Staff/containers/ActivityForm.tsx

```diff
@@ -100,23 +99,29 @@ const ActivityForm = ({ activity, activityClassification, contentPartnerOptions,
     handleAttributeChange('topic_ids', options)
   }
 
-  const flagOptionElements = flagOptions.map(fo => (<option value={fo}>{fo}</option>))
+  const initialFlag = editedActivity.flags[0]
+  const flagOptionElements = flagOptions.map(fo => {
+    if (fo === initialFlag) {
+      return <option key={fo} selected value={fo} >{fo}</option>
+    }
+    return <option key={fo} value={fo} >{fo}</option>
+  })
 
-  const standardOptionElements = standardOptions.map(so => (<option value={so.id}>{so.name}</option>))
+  const standardOptionElements = standardOptions.map(so => (<option key={so.id} value={so.id}>{so.name}</option>))
 
-  const activityCategoryOptionElements = activityCategoryOptions.map(so => (<option value={so.id}>{so.name}</option>))
+  const activityCategoryOptionElements = activityCategoryOptions.map(so => (<option key={so.id} value={so.id}>{so.name}</option>))
 
   let followUpActivityField
-
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Disabling these 2 rules because:
- this PR did not cause the violations, just uncovered them
- removing the violations would [change the HTML structure, or something more invasive](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md), without providing much value

Reply:
I don't think we should disable these rules here. They're only staged file linter rules, so they're not going to make your PR unmergeable, but they are a standard we'd like to hold to. I think it's fine to ignore it for now if you'd like to, since you didn't introduce the issue, but the fix is just adding an id to the `select` element and an `htmlFor={[selectElementId]}` to the `label` element.